charts/alertmanager/templates/_helpers.tpl

```diff
@@ -43,7 +43,7 @@ Image name.
 {{- end -}}
 
 {{ define "alertmanager.url" -}}
-alertmanager.{{ .Values.global.baseDomain }}
+{{ .Values.global.prefixDomain }}-alertmanager.{{ .Values.global.baseDomain }}
```

Comment:
Why are we forcing users into a prefix mechanism rather than letting them specify an arbitrary domain name?

Reply:
Houston as this regex for subdomain to check string that starts with these names
**/^(grafana|kibana|alertmanager|prometheus)$** 
https://github.com/astronomer/houston-api/blob/main/src/routes/v1/authorization/handler.js#L24
Lets discuss this today...and can tell Alex, based on which they can change..

Reply:
Would it make sense for this regex /^(grafana|kibana|alertmanager|prometheus)$/ to be configurable? It could default to the current value but allow someone to modify it so they can use w/e domains they want?